app/models/manageiq/providers/awx/provider.rb

```diff
@@ -79,6 +80,16 @@ def self.params_for_create
     }.freeze
   end
 
+  def self.url_format_valid?(url)
+    URI.regexp(%w[http https]).match?(url)
+  end
+
+  def url_format_valid?
+    return unless url.present? # Presence is checked elsewhere
+
+    errors.add(:url, N_("has to be a valid URL")) unless self.class.url_format_valid?(url)
```

Comment:
Should this be translated immediately, or does something else translate it later?


```suggestion
    errors.add(:url, _("has to be a valid URL")) unless self.class.url_format_valid?(url)
```